style.css

```diff
@@ -19,6 +23,75 @@
     padding-bottom: 10px;
   }
 
+  #menu-navbar {
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    background-color: rgba(96, 112, 255, 0.9);
+    z-index: 999;
+    display: flex;
+    flex-direction: column;
+    list-style: none;
+    padding: 20px;
+  }
```

Comment:
- [ ] Please ensure to have a blurred background color. The app contents should not be clear. This is to ensure that we stick to the Figma template design as much as possible and also have a professional portfolio design

Your design 😎 
![image](https://github.com/HtetWaiYan7191/HtetWaiYan7191.github.io/assets/99394374/2d5bc910-7241-4d35-97ae-c1bc7ed3dfb1)

Figma design 😎 
![image](https://github.com/HtetWaiYan7191/HtetWaiYan7191.github.io/assets/99394374/89a340d8-267a-45ed-85fe-ee04e7e8114a)

Reply:
Thank you very much for clear instructions 😁 🙇